examples/headsail-bsp/sdram_hpc.x

```diff
@@ -1,12 +1,12 @@
 MEMORY
 {
   BOOTRAM : ORIGIN = 0x10000, LENGTH = 0x8000
```

Comment:
This `BOOTRAM` line should be removed if it's not used in the rest of the script.

Reply:
`BOOTRAM` removed